packages/origin-ui-core/src/components/Account/UserProfile/UserDataEditForm.tsx

```diff
@@ -0,0 +1,260 @@
+                                            )}
+                                            className="mt-3"
+                                        />
+                                    ) : (
+                                        <Button
+                                            type="button"
+                                            variant="contained"
+                                            color="primary"
+                                            className="mt-4 mb-2 right"
+                                            onClick={async () => {
+                                                const {
+                                                    data: { success }
+                                                } = await userClient.reSendEmailConfirmation();
+
+                                                const message = t(
+                                                    success
+                                                        ? 'user.feedback.confirmationEmailResent'
+                                                        : 'user.feedback.confirmationEmailResentFailed'
+                                                );
+
+                                                showNotification(
+                                                    message,
+                                                    success
+                                                        ? NotificationType.Success
+                                                        : NotificationType.Error
+                                                );
+                                            }}
+                                        >
+                                            {t('user.actions.resendConfirmationEmail')}
+                                        </Button>
+                                    )}
+
+                                    <Field
+                                        label={t('user.properties.kycStatus')}
+                                        type="kycStatus"
+                                        name="kycStatus"
+                                        component={TextField}
+                                        variant="filled"
+                                        fullWidth
+                                        disabled={true}
+                                        value={values.kycStatus}
+                                        className="mt-3"
+                                    />
+                                </Grid>
+                            </Grid>
+                            {isEditing && (
+                                <Button
+                                    style={{ marginRight: 10 }}
+                                    type="button"
+                                    variant="contained"
+                                    color="primary"
+                                    className="mt-3 right"
+                                    disabled={
```

Comment:
After first change of data Save button is no longer disabled.

Reply:
fixed
saving with success or canceling should now quit editing mod, then re-entering should keep save button disabled initially.